ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml

```diff
@@ -22,40 +22,268 @@ resources:
       memory: 200Mi
 tests:
 - as: ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2
-  cron: 0 12 * * 4
+  cron: 0 12 * * 1
   steps:
     cluster_profile: aws-qe
     env:
       CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
       OPENSHIFT_VERSION: "4.13"
+      REPLICAS: "3"
     test:
     - ref: openshift-qe-cluster-density-v2
     workflow: rosa-aws-sts
-- as: ocp-qe-perfscale-ci-tests-rosa-node-density
+- as: ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2-24nodes
+  cron: 0 12 * * 2
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.13"
+      READY_WAIT_TIMEOUT: 30m
+      REPLICAS: "24"
+    test:
+    - ref: openshift-qe-cluster-density-v2
+    workflow: rosa-aws-sts
+- as: ocp-qe-perfscale-ci-tests-rosa-hcp-cluster-density-v2-24nodes
+  cron: 0 12 * * 3
+  steps:
+    cluster_profile: aws-sd-qe
```

Comment:
is this this the right profile? do we know what the difference is between this and aws-qe?

Reply:
It is ok to use either. Both are here to stay. I don't know the history of both, but I confirmed with functional QE that it's ok to use either.

Reply:
should we stay consistent with using aws-qe though? I only see 2 of the many tests you added here using this profile

Reply:
It doesn't really matter, but on a 2nd thought I am thinking of leaving this inconsistency here on purpose. If we find any issues in the future because of this difference, it will give us more insights to adapt.

Reply:
I think we should keep consistent throughout and then if we hit problems know that there is another profile available. but think this profile we can request more quota if we hit issues